codecept.conf.js

```diff
@@ -19,8 +19,8 @@ exports.config = {
       url: '',
       show: true,
       chrome: {
-        //args: ['--headless', '--no-sandbox', '--window-size=1536,826', '--ignore-certificate-errors'],
-        args: ['--no-sandbox', '--window-size=1536,826','--ignore-certificate-errors'],
+        args: ['--headless', '--no-sandbox', '--window-size=1536,826', '--ignore-certificate-errors'],
+        //args: ['--no-sandbox', '--window-size=1536,826','--ignore-certificate-errors'],
```

Comment:
Suggestion, maybe for another time, but you could pass in --headless in the environment variables, so this would become:

```suggestion
        //args: [process.env.headless, '--no-sandbox', '--window-size=1536,826','--ignore-certificate-errors'],
```

then, the env variable in the OS could be '--headless' or ''

Reply:
Done